layouts/partials/events/render-event.html

```diff
@@ -72,11 +74,10 @@ <h2>Examples</h2>
     */}}
     {{ if $desired_example_name }}
         {{ if eq $example_name $desired_example_name }}
-            {{ $example_content := partial "json-schema/resolve-refs" (dict "schema" $example "path" "event-schemas/examples") }}
-```json
-{{ jsonify (dict "indent" "  ") $example_content }}
-```
+            {{ partial "events/example" (dict "example" $example)}}
         {{ end }}
+    {{ else if eq $event_name $example_name }}
+        {{ partial "events/example" (dict "example" $example)}}
```

Comment:
Looks like this is to cover the case that `event_name` has a `$`, so will never be matched by the case below? Could you add a comment to say so?

Reply:
(assuming you keep the case below)